backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainResults.java

```diff
@@ -0,0 +1,18 @@
+package com.moh.phlat.backend.esb.json;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class MaintainResults {
```

Comment:
since all of them need lombock, better to put it at class level

Reply:
This one I will change to class level